image/discourse_test/install-chrome

```diff
@@ -10,5 +10,5 @@ else
   wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add - &&\
     echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list &&\
     apt update &&\
-    apt install -y google-chrome-stable chromium-driver
```

Comment:
chromium is causing all sorts of problem on Debian bookworm so we don't need to install it.

Reply:
What problems were you seeing?

It's still installed in the dev image, and it's kinda essential if we want people to have a decent experience developing on arm64. So just wondering what problems we should look out for?